whipper/command/cd.py

```diff
@@ -455,7 +461,16 @@ def _ripIfNotRipped(number):
                                     self.ittoc.getTrackLength(number), number)
 
             self.program.saveRipResult()
-
+        try:
+            excluded_tracks = self.options.excluded_tracks.split(",")
+        except UnboundLocalError:
+            excluded_tracks = []
+        excluded_tracks_integer = []
```

Comment:
I think we should keep the array of excluded tracks in `options.excluded_tracks`.